stack.yaml

```diff
@@ -21,10 +21,11 @@ extra-deps:
   commit: 9059edbd30c7d3b07cd97306158c4f2f33bb7a44
   subdirs:
     - yamlparse-applicative
-- github: NorfairKing/safe-coloured-text
-  commit: 9db3defadd2d2f05b2db02b551c6458fc4d7c10b
+- github: notquiteamonad/safe-coloured-text
```

Comment:
This will need changing to a proper reference to NorfairKing/safe-coloured-text once a commit with this split exists.